ova/bootable/build-app.sh

```diff
@@ -34,17 +34,23 @@ chage -I -1 -m 0 -M 99999 -E -1 root
 
 log3 "configuring ${brprpl}UTC${reset} timezone"
 ln --force --symbolic /usr/share/zoneinfo/UTC /etc/localtime
-log3 "configuring ${brprpl}en_US.UTF-8${reset} locale"
-/usr/bin/touch /etc/locale.conf
-/bin/echo "LANG=en_US.UTF-8" > /etc/locale.conf
-/sbin/locale-gen.sh
+
+## TODO: This causes issues on photon 2.0, debug or remove permanently when root cause
```

Comment:
Should we file as issue to track this? (I'm unsure of the Dispatch conventions here, but that may make it easier to associate this with VIC; they'll be looking at Photon 2.0 for 1.5.)

Reply:
Sounds good, thanks for reminding me!

Reply:
You can try to install glibc-i18n package.